app/utils/BFS.js

```diff
@@ -1,122 +1,120 @@
-const pathFind = (field, startP, endP) => {
-	const path = [];
+const findPath = (field, startP, endP) => {
+  const path = [];
 
-	// tests if the point is inside the tested field				
-	var inField = function (point) {
-		if (point[0]<field.length && point[0] >=0) {
-			if (point[1]<field[0].length && point[1] >=0) {
-				return true;
-			}
-		}
-		return false;
-	}
+  // tests if the point is inside the tested field
+  const inField = function (point) {
+    if (point[0] < field.length && point[0] >= 0) {
```

Comment:
Name this condition. Either in a variable or function. You want to read what the code does, not how it's implemented.